opm/simulators/linalg/bda/BILU0.cpp

```diff
@@ -150,6 +150,118 @@ bool BILU0<block_size>::analyze_matrix(BlockedMatrix *mat)
     return true;
 } // end init()
 
+template <unsigned int block_size>
+bool BILU0<block_size>::analyze_matrix(BlockedMatrix *mat, BlockedMatrix *jacMat)
```

Comment:
Can't we get along with one analyze_matrix method and some if statements? This seems to be a lot of duplicate code.